### PR TITLE
SC-2139 First cut for location open-api-v3 model transformation

### PR DESCRIPTION
**Pull request by Hari-stackroute**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://628.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://628.qa.docs.sunbird.org/api-documentation/apis/locationapi// <br />  <br/> **Modified sections are -** <br />

**Review by Sunitashelar — CHANGES_REQUESTED** [review]

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://628.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://628.qa.docs.sunbird.org/api-documentation/apis/locationapi// <br />  <br/> **Modified sections are -** <br />

File: apis/locationapi/locationapi.yaml [hunk 1]
```diff
@@ -1,617 +1,1290 @@
...
+          - The endpoint for **Create location** is `/create`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+      operationId: Create-Location
       parameters:
       - name: Authorization
         in: header
-        description: '''To make use of any User API, you require authorization. Raise
-          a request to the administrator for the use of the API. You will receive
-          the authorization key. Specify the key received, here.'''
+        description: 'To make use of any User-Feed API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply apis/locationapi/locationapi.yaml]: Changes done @Sunitashelar

File: apis/locationapi/locationapi.yaml [hunk 2]
```diff
@@ -1,617 +1,1290 @@
...
+           - The endpoint for **Search location** is `/search`
+           - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+      operationId: Search-Location
       parameters:
       - name: Authorization
         in: header
-        description: To make use of any location API, you require authorization. Raise
-          a request to the administrator for the use of the API. You will receive
-          the authorization key. Specify the key received, here.
+        description: 'To make use of any User-Feed API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
```
[the remark this answers is not in the document]
Comment (Hari-stackroute) [reply apis/locationapi/locationapi.yaml]: Changes done @Sunitashelar

File: apis/locationapi/locationapi.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sunitashelar) [reply apis/locationapi/locationapi.yaml]: Done

**Review by Sunitashelar — APPROVED** [review]

**Pull request merged**